guides/release/components/built-in-components.md

```diff
@@ -1,43 +1,42 @@
-Out of the box, Ember provides 2 components for building a form:
+Ember fournit deux composants prêts à l'emploi pour construire des formulaires:
 
 * [`<Input>`](https://api.emberjs.com/ember/release/classes/Ember.Templates.components/methods/Input?anchor=Input)
 * [`<Textarea>`](https://api.emberjs.com/ember/release/classes/Ember.Templates.components/methods/Textarea?anchor=Textarea)
 
-These components are similar in HTML markup to the native `<input>` or `<textarea>` elements. In contrast to the native elements, `<Input>` and `<Textarea>` automatically update the state of their bound values.
-
+Le markup HTML de ces composants est similaire à celui des éléments `<input>` et `<textarea>`. Au contraire de ces éléments natifs, `<Input>` et `<Textarea>` mettent automatiquement à jour l'état des valeurs qui leur sont liées.
 
 ## `<Input>`
 
-We mentioned that the built-in components are similar in HTML markup to their native counterparts. What does this mean?
+Nous avons dit plus haut que les composants _built-in_ (c'est-à-dire "construits à l'intérieur du framework") ont un markup HTML similaire à celui de leur équivalent natif. Que'est-ce que ça signifie ?
 
-Consider the following example in a template file.
+Considérons l'exemple suivant, le template d'un fichier.
 
 ```handlebars
-<label for="user-question">Ask a question about Ember:</label>
+<label for="user-question">Posez une question sur Ember :</label>
 <Input
   id="user-question"
   @type="text"
-  @value="How do text fields work?"
+  @value="Comment fonctionnent les champs textes ?"
 />
 ```
 
-When Ember renders this template, you will see the following HTML code:
+Quand Ember rend ce template, vous pouvez voir le code HTML suivant : 
 
 ```html
-<label for="user-question">Ask a question about Ember:</label>
-<input id="user-question" type="text" value="How do text fields work?" />
+<label for="user-question">Posez une question sur Ember :</label>
+<input id="user-question" type="text" value="Comment fonctionnent les champs textes ?" />
 ```
 
 
-### Ways to associate labels and inputs
+### Associer _labels_ et _inputs_
 
-Every input should be associated with a label. In HTML, there are a few ways to do this. With the built-in `<Input>` component,
+Chaque _input_ (champ texte) doit être associé à un _label_ (étiquette). Le composant built-in `<Input>` permet de :
```

Comment:
> In HTML, there are a few ways to do this.

I think we should keep this, as it "explains" why we have a list after ^^

Reply:
```suggestion
Chaque _input_ (champ texte) doit être associé à un _label_ (étiquette). Il y a différentes manières d'écrire cette association en HTML. Le composant built-in `<Input>` permet de :
```